crates/biome_service/src/settings.rs

```diff
@@ -494,20 +494,25 @@ impl From<PartialJsonConfiguration> for LanguageSettings<JsonLanguage> {
     }
 }
 
-impl From<CssConfiguration> for LanguageSettings<CssLanguage> {
-    fn from(css: CssConfiguration) -> Self {
+impl From<PartialCssConfiguration> for LanguageSettings<CssLanguage> {
+    fn from(css: PartialCssConfiguration) -> Self {
         let mut language_setting: LanguageSettings<CssLanguage> = LanguageSettings::default();
 
-        language_setting.parser.allow_wrong_line_comments = css.parser.allow_wrong_line_comments;
-        language_setting.parser.css_modules = css.parser.css_modules;
-
-        language_setting.formatter.enabled = Some(css.formatter.enabled);
-        language_setting.formatter.indent_width = css.formatter.indent_width;
-        language_setting.formatter.indent_style = css.formatter.indent_style.map(Into::into);
-        language_setting.formatter.line_width = css.formatter.line_width;
-        language_setting.formatter.line_ending = css.formatter.line_ending;
-        language_setting.formatter.quote_style = Some(css.formatter.quote_style);
-        language_setting.linter.enabled = Some(css.linter.enabled);
+        if let Some(parser) = css.parser {
+            language_setting.parser.allow_wrong_line_comments = parser.allow_wrong_line_comments;
+            language_setting.parser.css_modules = parser.css_modules;
+        }
+        if let Some(formatter) = css.formatter {
+            language_setting.formatter.enabled = Some(formatter.enabled.unwrap_or_default());
```

Comment:
I wonder if we can add a TODO comment here, in case we forget where to change when we want to enable it 🤦